modules/storage-azure-file-create-storage-class.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent-storage-azure-file.adoc
+
+[id="storage-create-azure-file-storage-class_{context}"]
+= Creating the Azure storage class
+
+StorageClasses are used to differentiate and delineate storage levels and
+usages. By defining a storage class, users can obtain dynamically provisioned
+persistent volumes.
+
+.Additional References
+
+* https://kubernetes.io/docs/concepts/storage/storage-classes/#azure-file[Azure File Storage Class]
+
+.Procedure
+
+. In the {product-title} console, click *Storage* -> *Storage Classes*.
+
+. In the storage class overview, click *Create Storage Class*.
+
+. Define the desired options on the page that appears.
+
+.. Enter a name to reference the storage class.
+
+.. Enter an optional description.
+
+.. Select the reclaim policy.
+
+.. Select `kubernetes.io/azure-file` from the drop down list.
+
+... Optionally, enter the SKU name. This corresponds to your Azure
+storage account SKU tier. Valid options are `Premium_LRS`, `Standard_LRS`,
+`StandardSSD_LRS`, and `UltraSSD_LRS`.
+
+... Optionally, enter the location, or region, of the storage account, such as `eastus`.
+
+... Optionally, define the storage account name. If the storage account name
+is defined, then {product-title} will ignore the prior two options and only
```

Comment:
s/will ignore/ignores
s/provision/provisions

I'd make the two values that it ignores explicit.